tests/runtime_neutral/test_claude_preview_scaffold.py

```
@@ -196,6 +196,43 @@ def test_preview_check_accepts_preview_settings_file_without_touching_real_setti
+            powershell,
+            '-NoProfile',
+            '-File',
+            str(REPO_ROOT / 'install.ps1'),
+            '-HostId',
+            'claude-code',
+            '-TargetRoot',
+            str(self.target_root),
+            '-Profile',
+            'full',
+        ]
+        subprocess.run(install_cmd, capture_output=True, text=True, check=True)
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)